Sources/NetworkService/NetworkError.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Contains network errors that may occur, while the application is running.
+public enum NetworkError: Error, Codable, CaseIterable {
+    /// Indicates, that the Internet connection has been lost.
+    case lostConnection
+    
+    /// Indicates, that the server cannot handle the request (because it is overloaded or down for maintenance).
+    case serverUnavailable
+    
+    /// Indicates, that the server can not or will not process the request due to an apparent client error
+    /// (e.g., malformed request syntax, size too large, invalid request message framing, or deceptive request routing).
+    /// or
+    /// The request could not be understood by the server due to invalid syntax.
+    case badRequest
+    
+    /// Indicates, that the client needs to authenticate to gain network access.
+    case authenticationError
+    
+    /// Indicates, that the request to connect and log on to an FTP server could not be completed
+    /// because the supplied user name is incorrect.
+    case userNameError
```

Comment:
Why do we need this error?

Reply:
Agree, we don't need it in our app. The case is removed.
Thanks.